resources/function-controller/values.yaml

```diff
@@ -8,15 +8,19 @@ tests:
   enabled: true
   image:
     repository: "eu.gcr.io/kyma-project/function-controller-test"
-    tag: "PR-7581"
+    tag: "PR-8043"
     pullPolicy: IfNotPresent
-  disableConcurrency: true
+  disableConcurrency: false
   restartPolicy: Never
   resources:
+    requests:
+      memory: 32Mi
+      cpu: 100m
     limits:
-      memory: 128Mi
+      memory: 64Mi
+      cpu: 200m
   envs:
-    waitTimeout: 45m # sorry guys :sadgabens:
+    waitTimeout: 5m
```

Comment:
I would not be so brave here 😛

Reply:
As you probably noticed, it passed on 4 pipelines :P For minikube there will proper override.